io-uring-test/src/tests/net.rs

```diff
@@ -687,7 +687,7 @@ pub fn test_tcp_accept_multi<S: squeue::EntryMarker, C: cqueue::EntryMarker>(
     assert_eq!(cqes[op1].user_data(), 2002);
     assert_eq!(cqes[op2].user_data(), 2003);
 
-    assert_eq!(cqes[op1].result(), -125); // -ECANCELED
+    assert_eq!(cqes[op1].result(), -libc::ECANCELED); // -ECANCELED
```

Comment:
The trailing doc comment is a bit redundant now - could be removed

Reply:
Addressed